Lib/distutils/command/install.py

```diff
@@ -223,7 +223,7 @@ def initialize_options(self):
 
     def finalize_options(self):
         """Finalizes options."""
-        # This method (and its pliant slaves, like 'finalize_unix()',
+        # This method (and its pliant childs, like 'finalize_unix()',
```

Comment:
I think this reads better as `children`. Although `pliant` isn't a very common word, and I'm not sure how it's being used here.

Reply:
Oh, maybe I should have waited longer before merging this one :-( I'm wasn't 100% sure between "childs" and "children". Sorry, english is not my first language. I created PR #9102 to replace childs with children.

I'm not sure what do you propose for pliant. In case of doubt, I leave it unchanged.

Reply:
@gvanrossum, @ericvsmith: Ok, I created https://github.com/python/cpython/pull/9195

Reply:
@gvanrossum it's in the same vein as compliant.